src/adapter/src/session/vars.rs

```diff
@@ -399,6 +402,7 @@ static EMIT_TRACE_ID_NOTICE: ServerVar<bool> = ServerVar {
 #[derive(Debug)]
 pub struct SessionVars {
     application_name: SessionVar<str>,
+    build_info: &'static BuildInfo,
```

Comment:
Should this go on SystemVars instead?

Reply:
System vars can be set with `ALTER SYSTEM SET` which is not what we want here. It's kinda a funny split (probably system vars and server/session vars should be unified somehow), but for now this is the right spot!